Stream/Basic/MyStreamProcessor.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Castle.Core.Logging;
```

Comment:
I'm not familiar with this one, is this using intentional?

Reply:
fixed. thanks!